src/components/Book.js

```diff
@@ -1,11 +1,17 @@
 import React from "react"
+import { Button } from "react-bootstrap"
+import { removeBook } from "../actions"
+import { useDispatch } from "react-redux"
 
 const Book = ({ book }) => {
+  const dispatch = useDispatch()
+
   return (
     <ul>
       <li>
-        Book ID:{book.id} - Book title:{book.title} - Book category:
+        Book ID: {book.id} - Book title: {book.title} - Book Category:{" "}
         {book.category}
+        <Button onClick={() => dispatch(removeBook(book))}>Delete Book</Button>
       </li>
     </ul>
   )
```

Comment:
## Component Implementation 
* [ ] This component should add `Remove book` button to the book row [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/RemoveBookButton.png)
* [ ] It should be attaching passed `handleRemoveBook` to on click event of `Remove book` button
* [ ] It should be staying as a presentational component

#### Kindly Implement these changes as they are mandatory, team!